plugins/services/src/js/containers/pod-instances/PodInstancesTable.js

```diff
@@ -138,6 +138,8 @@ class PodInstancesTable extends React.Component {
   }
 
   getColumns() {
+    // TODO: add sort function here
```

Comment:
@DanielMSchmidt the sorting methods shouldn't be written in this method. You can follow what we have at `https://github.com/dcos/dcos-ui/blob/f0e9c4425538c107bebb8b06784ab914a8be07b9/plugins/services/src/js/containers/tasks/TaskTable.js#L77`